docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -94,14 +94,21 @@ Add the config plugin to your [app config](/versions/latest/config/app/#plugins)
 
 After embedding the font with the config plugin, create a [new development build](/develop/development-builds/create-a-build/) and install it on your device or Android Emulator or iOS Simulator.
 
-You can use that font on `<Text>` by using `fontFamily` style prop. For example, the `<Text>` uses the `Inter-Black` as the font family:
+You can use the font with `<Text>` by specifying the `fontFamily` style prop. [Read below](#how-to-determine-which-font-family-name-to-use) on how to determine the font family. For example:
```

Comment:
added a link here because looking at the example, many people would wrongly assume that that file name == font family and stop reading further